README.md

```diff
@@ -46,7 +46,7 @@ npm install sparkpost
     * `options` - [see request modules options](https://github.com/mikeal/request#requestoptions-callback)
     * `options.uri` - can either be a full url or a path that is appended to `options.origin` used at initialization ([url.resolve](http://nodejs.org/api/url.html#url_url_resolve_from_to))
     * `options.debug` - setting to `true` includes full response from request client for debugging purposes
-    * `callback` - executed after task is completed. **required**
+    * `callback` - executed after task is completed if provided
```

Comment:
Can we add a bit to these sections that specify what the method returns (ie a promise) to help clarify this?